niworkflows/viz/utils.py

```diff
@@ -199,20 +199,45 @@ def extract_svg(display_object, dpi=300, compress='auto'):
 def cuts_from_bbox(mask_nii, cuts=3):
     """Finds equi-spaced cuts for presenting images"""
     from nibabel.affines import apply_affine
-    mask_data = mask_nii.get_data()
-    B = np.argwhere(mask_data > 0)
-    start_coords = B.min(0)
-    stop_coords = B.max(0) + 1
+
+    mask_data = mask_nii.get_data() > 0.0
+
+    # First, project the number of masked voxels on each axes
+    ijk_counts = [
+        mask_data.sum(2).sum(0),  # project to i (sagittal)
+        mask_data.sum(2).sum(1),  # project to j (coronal)
+        mask_data.sum(1).sum(0),  # project to k (axial)
```

Comment:
And just to be consistent:

```suggestion
        mask_data.sum(1).sum(0),  # project axial planes to vertical (k) axis
```